vtysh/vtysh.c

```diff
@@ -474,7 +474,7 @@ int vtysh_mark_file(const char *filename)
 	}
 
 	vty = vty_new();
-	vty->fd = 0; /* stdout */
+	vty->wfd = STDERR_FILENO;
```

Comment:
Can't you use `stderr` here instead of `STDERR_FILENO`?

Reply:
My understanding is that vty does unbuffered I/O with a file descriptor. 

`STDERR_FILENO` is the file descriptor for stderr, while `stderr` is a libc buffered I/O wrapper around that. 

So I am not sure how I would use `stderr` if not like `fileno(stderr)`.